packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -53,6 +64,56 @@ const getInputElementForType = (type: FormControlProps['type']) => {
   return CustomInputs[customInput] || Input;
 };
 
+function useFormTextAnimation() {
+  const prefersReducedMotion = usePrefersReducedMotion();
+
+  const getFormTextAnimation = useCallback(
+    (enterAnimation: boolean): ThemableCssProp => {
+      if (prefersReducedMotion) {
+        return {
+          animation: 'none',
+        };
+      }
+      return t => ({
+        animation: `${enterAnimation ? animations.inAnimation : animations.outAnimation} 300ms ${
+          t.transitionTiming.$common
+        }`,
+        transition: 'height 200ms ease', // This is expensive but required for a smooth layout shift
+      });
+    },
+    [prefersReducedMotion],
+  );
+
+  return {
+    getFormTextAnimation,
+  };
+}
+
+const useCalculateErrorTextHeight = () => {
+  const [height, setHeight] = useState(24);
```

Comment:
❓ What's this unit? Is it pixels?

Reply:
Yes

Reply:
We can make this more apparent by making a type alias, `type Px = number` and typing the hook as such: 
```
  const [height, setHeight] = useState<Px>(24);
```